### ISSUE TITLE
Change descriptions of sweater areas

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#227]
In #215 @terracoda said

> "At top right arm", better as "On right shoulder." @terracoda to provide updated sweater locations.

@terracoda if updated descriptions are necessary can you please provide a list changes?

**Comment by terracoda** [comment]
Adding dots for shoulders on the Play Area Grid diagram. I am thinking when the balloon is at the top (i.e., can't go any further) in the arm columns, the description should be "On left shoulder" and "On right shoulder", instead of "At top upper left arm" and "At top upper-right arm." When the balloon is not in the top position (i.e., shoulder position), the description can be "On upper-left arm" or "On upper-right arm."

I will review the examples and see how this works.
![bs-grid-new-with-shoulders](https://cloud.githubusercontent.com/assets/1902116/21404189/9101b9e6-c79a-11e6-9617-38d2c23b9759.png)

**Comment by terracoda** [comment]
@jessegreenberg, what do you think of a simplification to the grid? If the upper and lower regions are smaller, we can make the general area with no defined "upper" and "lower" adjectives. I think this will improve things and should streamline phrasing as well, but I am still assessing.
![bs-grid-new-adjusted-upper-lower-regions](https://cloud.githubusercontent.com/assets/1902116/21409001/8f13406a-c7b2-11e6-8485-d1fd77884f5c.png)

For example, If the upper and lower regions are smaller, we don't have to say upper and lower as often and if the _at edge, at top, at bottom_ are treated more like markers rather than positional information, we may be able to simplify further.

MOVING UP or DOWN in far left column 
**Dynamic Descriptions**
_Sticking to:_
- top left edge of sweater -> left shoulder of sweater. At top (assuming Upward motion at boundary).
- upper left edge of sweater -> left arm of sweater.
- upper left edge of sweater -> left arm of sweater.
- lower left edge of sweater -> lower-left arm of sweater.
- bottom left edge of sweater -> lower-left arm of sweater.

**Simultaneous Interaction Alerts for position on sweater:**
_Up. [+charge info]. On:_
- top left edge of sweater. -> left shoulder of sweater. At top.
- upper left edge of sweater. -> left arm of sweater.
- upper left edge of sweater. -> left arm of sweater.
- lower left edge of sweater. -> lower-left arm of sweater.
- bottom left edge of sweater. -> lower-left arm of sweater.

Then perhaps, at edge, at top, and at bottom, are only added if the user hits boundary, like a spearcon (speech earcon).

Note that in the interaction alerts, I think we may drop "of sweater" after sometime??
The release descriptions should then be very close if not identical to the dynamic description.

**Comment by terracoda** [comment]
@jessegreenberg, would you be able to put the updated grid we discussed yesterday (with no changes in descriptions) into the accessibility-test? And also provide a parameter for the simpler grid (again no change in phrase descriptions). This will help me assess if the differences and give you a full list of any required changes in phrasing.

**Comment by emily-phet** [comment]
@terracoda I like your proposed simpification. I think out of context, the original descriptions sound more complete, but in context (when a user is actively exploring the sweater) the proposed changes will likely make plenty of sense, be more direct, and sound more like natural language.

**Comment by terracoda** [comment]
@emily-phet, great. I'm downloading my google sheets so I can work on them while I travel.

**Comment by terracoda** [comment]
In preparation to simplify the grid descriptions (including sweater locations) I have copied the old grid descriptions into a new google sheet called [Grid Descriptions Simplified and Revised](https://docs.google.com/spreadsheets/d/1fZq21j65jfJhcsA22pasuBA00kM-cGWxsWDsogwhC64/edit#gid=0)

I haven't changed any descriptions yet, just added the new grid diagram.

**Comment by terracoda** [comment]
@jessegreenberg and @emily-phet, there are twelve locations for the sweater, 9 for the Play Area, and three for the wall:
**12 sweater locations:**
- left shoulder, left arm, lower-left arm
- upper-left side, left side, lower-left side
- upper-right side, right side, lower right-side
- right shoulder, right arm, lower-right arm

**9 Play Area locations:**
- upper-left side, left side, lower-left side
- upper-center, center, lower-center
- upper-right side, right side, lower-right side

**3 wall locations:**
- upper
- NO ADJECTIVE
- lower

**Example location strings for a balloon's Dynamic Description (DD):**
- Sticking to left shoulder of sweater.
- Sticking to lower-right side of sweater.
- In upper-left side of Play Area. 
- In left side Play Area.
- Sticking to wall.
- Sticking to lower wall.

**Example location strings for Grab IA:**
- [[+ Indication of Grab]]. Sticking to upper-right side of sweater.
- [[+ Indication of Grab]]. Touching upper wall.
- [[+ Indication of Grab]]. In right side of Play Area.
- [[+ Indication of Grab]]. Sticking to wall.

**Example location strings Scene Summary:**
- Yellow balloon, sticking to right arm.
- Yellow balloon, sticking to lower-right arm.
- Yellow balloon, in upper-right side of Play Area.
- Yellow balloon, touching upper wall.
- Yellow balloon, in right side of Play Area.
- Yellow balloon, touching wall.
- Yellow balloon, sticking to wall.

Working on updating mini-cognitive walkthroughs in the google sheet [CogWT: BASE: New Grid Descriptions](https://docs.google.com/spreadsheets/d/1fZq21j65jfJhcsA22pasuBA00kM-cGWxsWDsogwhC64/edit#gid=0)

**Comment by terracoda** [comment]
@jessegreenberg, the old 18 sweater locations have been replaced by 12:
- top left edge & top left arm: left shoulder
- upper left edge & upper left arm: left arm (see visual model)
- lower left arm & bottom left arm: lower-left arm
- top left side & upper left side: upper-left side
- upper left side: left side (see visual model)
- lower left side & bottom left side: lower-left side
- top right side & upper right side: upper-right side 
- upper right side: right side (see visual model)
- lower right side & bottom right side: lower-right side
- top right arm & upper right arm: upper-right arm
- upper right arm: right arm (see visual model)
- lower right arm & bottom right arm: lower-right arm

These changes are also noted in google doc.

**Comment by terracoda** [comment]
This version of the grid has a small up date. It clearly labels the "General Area" that requires no adjectives on locations, and repalces the spearcons on the shoulders with "At top." The actual locations here are "left shoulder" and "on right shoulder".
![bs-grid-with-spearcons](https://cloud.githubusercontent.com/assets/1902116/22965935/f323cf26-f385-11e6-9a29-d27a9269f2c7.png)

**Comment by terracoda** [comment]
@jessegreenberg, I think it would be best to first replace the sweater locations. I think this would be straight forward as I have not changed the actual phrases for rubbing and finding charges. 

Would you agree?

**Comment by terracoda** [comment]
@jessegreenberg, I created a branch called "description-string-updates," and updated several strings including sweater locations. Let me know (no hurry) if you see the branch I created?

**Comment by terracoda** [comment]
@jessegreenberg, pull request #254 containing the description-string-updates is ready for review.

**Comment by jessegreenberg** [comment]
We have updated the grid and associated descriptions as part of a major overhaul of designs and implementation for BASE a11y that started a while ago.  Closing issue as the sim is up to date with the latest design docs.

**Comment by terracoda** [comment]
Awesome work @jessegreenberg !
